task-services/src/test/java/com/taat/taskservices/services/comparators/TaskDurationComparatorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.taat.taskservices.services.comparators;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.junit.jupiter.api.Assertions;
+
+import com.taat.taskservices.model.Task;
+import com.taat.taskservices.utils.Duration;
+
+@ExtendWith(MockitoExtension.class)
+public class TaskDurationComparatorTest {
+
+    @Test
+    public void testCompare() {
+        List<Task> taskList = new ArrayList<>();
+        taskList.add(new Task("1", "testOwner", "Task Short", "A short task", null, null, null, 5, Duration.S.toString(), false, false, Collections.emptyList()));
```

Comment:
This is a good, thorough test of the best-case scenario (the tasks are already in the order that we would want them to be sorted in the output) but we should have at least one test scenario where the tasks in the list are in a random order or even a reverse order to make sure that the comparator is working as intended.